### ISSUE TITLE
Error when pytest is attempted to run by itself

### ISSUE THREAD

**Issue by jeffwright13** [jeffwright13/pytest-tui#57]
Brand new venv with only pytest-tui installed.
```
python -m venv venv
source venv/bin/activate
pip install -i https://pypi.org ~/coding/pytest-tui/dist/pytest_tui-1.2.0-py2.py3-none-any.whl
```

`pytest --version` gives error:
```
$ pytest --version

pytest 7.1.2
sys:1: ResourceWarning: unclosed file <_io.BufferedRandom name=3>
(venv)
```

**Comment by jeffwright13** [comment]
Fixed
   f501d39..6068e77  issue53 -> issue53

**Comment by jeffwright13** [comment]
Apparently not fixed - still happening in 1.3.0:
```
$ pytest 
========================================== test session starts ===========================================
platform darwin -- Python 3.9.9, pytest-7.1.2, pluggy-1.0.0
rootdir: /Users/jwr003/reltest
plugins: Faker-14.1.0, tui-1.3.0, metadata-2.0.2
collected 0 items                                                                                        

========================================= no tests ran in 0.01s ==========================================
sys:1: ResourceWarning: unclosed file <_io.BufferedRandom name=3>
```

**Comment by jeffwright13** [comment]
Fixed in 1.3.3.
